src/stores/WalletStore.js

```diff
@@ -86,17 +86,13 @@ export default class WalletStore {
 
   @computed get lastAddressWithdrawLimit() {
     return {
-      QTUM: this.currentWalletAddress ? this.currentWalletAddress.qtum : 0,
-      BOT: this.currentWalletAddress ? this.currentWalletAddress.bot : 0,
+      QTUM: this.currentWalletAddress ? this.currentWalletAddress.naka : 0,
+      BOT: this.currentWalletAddress ? this.currentWalletAddress.nbot : 0,
     };
   }
 
   constructor(app) {
```

Comment:
add a reaction in the constructor to fetch all balances on new block number (GlobalStore.syncBlockNum)

Reply:
after looking at the NakaStore, we should move NakaStore -> WalletStore. dont see why we need to separate it unless you have a good reason?

Reply:
Just don't want to refactor too much now, want to keep change as minimal as possible to avoid errors. we can combine them after we launch the product.
And the reaction is already there in global store.